sdk/registry.go

```diff
@@ -1,11 +1,26 @@
 package sdk
 
 import (
+	"regexp"
+	"runtime"
 	"sync"
 
 	"github.com/flant/addon-operator/pkg/module_manager/go_hook"
 )
 
+// /path/.../global-hooks/a/b/c/Hook-name.go
+// $1 - Hook path
+// $3 - Hook name
+var globalRe = regexp.MustCompile(`/global[\/\-]hooks/(([^/]+/)*([^/]+))$`)
+
+// /path/.../modules/module-name/hooks/a/b/c/Hook-name.go
+// $1 - Hook path
+// $2 - module name
+// $4 - Hook name
+var moduleRe = regexp.MustCompile(`/modules/(([^/]+)/hooks/([^/]+/)*([^/]+))$`)
+
+var moduleNameRe = regexp.MustCompile(`^[0-9][0-9][0-9]-(.*)$`)
```

Comment:
I assume that this regex is incorrect because we have no rule for a module to start with a number.
Could you add a TODO here?

Reply:
Done.

Reply:
There is an issue about these "ugly hardcoded" numbers: https://github.com/flant/addon-operator/issues/39 (also #7)